src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -252,6 +256,7 @@ void getMany(RoutingContext routingContext) {
     final PatronGroupRepository patronGroupRepository = new PatronGroupRepository(clients);
 
     loanRepository.findBy(routingContext.request().query())
+      .thenApply(this::refuseWhenLoanWithoutItem)
```

Comment:
Do we only want to fix this in **getMany()** and not in **get()**?
Another question is, do we really want to fail during a call to **/loans/** if _one of the loans_ has a reference to non-existing item? And what happens now in this case? Would such call just fail with 500 status and NPE stacktrace?